docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
+Google Spreadsheet connector operations. This `<googlespreadsheet.init>` element used to authenticates  the user using OAuth2 authentication
+to access the Google account which contains the spreadsheets. For more information on authorizing requests in Google Spreadsheets, see [API Doc](https://developers.google.com/sheets/api/guides/authorizing).
+
+<br/>
+
+> Follow this [guide](https://docs.wso2.com/display/IntegrationCloud/Get+Credentials+for+Google+Spreadsheet) to set up the Google Spreadsheets and get credentials such as clientId, clientSecret, accessToken, refreshToken.
+
+<br/>
+
+**init**
+```xml
+<googlespreadsheet.init>
+    <accessToken>{$ctx:accessToken}</accessToken>
+    <clientId>{$ctx:clientId}</clientId>
+    <clientSecret>{$ctx:clientSecret}</clientSecret>
+    <refreshToken>{$ctx:refreshToken}</refreshToken>
+    <apiUrl>{$ctx:apiUrl}</apiUrl>
+</googlespreadsheet.init>
+```
+
+<br/>
+
+To get the OAuth access token directly call **init** (this method call getAccessTokenFromRefreshToken method itself )  method or add  `<googlespreadsheet.getAccessTokenFromRefreshToken>` element before `<googlespreadsheet.init>` element in your configuration.
```

Comment:
To directly get the OAuth access token, call the **init** method (this method call getAccessTokenFromRefreshToken method itself ) or add the `<googlespreadsheet.getAccessTokenFromRefreshToken>` element before the `<googlespreadsheet.init>` element in your configuration.